src/upload.ts

```diff
@@ -38,12 +35,7 @@ router.post('/upload', async (req, res) => {
         .pipe(transformer)
         .toBuffer();
 
-      const result = await Promise.any([
-        setFleek(buffer),
-        setInfura(buffer),
-        setPinata(buffer),
-        set4everland(buffer)
-      ]);
+      const result = await Promise.any(set(IMAGE_PROVIDERS, buffer));
```

Comment:
can we move `Promise.any` inside the `set` function?
```suggestion
      const result = await setProviders(buffer);
```